aiohttp_sse/__init__.py

```diff
@@ -37,47 +37,39 @@ def __init__(self, *, status=200, reason=None, headers=None):
         self.headers['Connection'] = 'keep-alive'
 
         self._loop = None
-        self._finish_fut = None
         self._ping_interval = self.DEFAULT_PING_INTERVAL
         self._ping_task = None
 
     def start(self, request):
+        if request.method != 'GET':
```

Comment:
Do we want to be backward compatible? we do not have even version of `0.1.0` so no promise to be backward compatible... what do you thin?